src/clj/chronograph/db/invite.clj

```diff
@@ -0,0 +1,35 @@
+(ns chronograph.db.invite
+  (:require [chronograph.db.core :as db]
+            [next.jdbc.sql :as sql]))
+
+(defn create!
+  ([organization-id email]
+   (create! db/datasource organization-id email))
```

Comment:
Let's remove this arity, now that we've decided to always require `tx`.

Reply:
Done